deployments/ansible/roles/signalfx-agent/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Check Signalfx Access Toekn is defined
+- name: Check SignalFx Access Token is defined
+  fail: msg="Bailing out. This play requires 'signalFxAccessToken'"
+  when: conf['signalFxAccessToken'] is undefined or conf['signalFxAccessToken'] is none or (conf['signalFxAccessToken'] | trim == '')
+
+# Import tasks according to the distribution
+- name: Import signalfx-agent deploy for CentOS or RHEL
+  import_tasks: yum_repo.yml
+  when: ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux' or ansible_distribution == 'Amazon'
```

Comment:
Try using the Jinja "in" operator as described at http://jinja.pocoo.org/docs/dev/templates/#other-operators so that this will be a bit easier to read.